src/h5web/vis-packs/nexus/NxValuesFetcher.tsx

```diff
@@ -21,6 +25,9 @@ function NxValuesFetcher(props: Props) {
     silxStyle,
   } = nxData;
 
+  usePrefetchValues([signalDataset, errorsDataset], dimMapping);
+  usePrefetchValues([...axisDatasets, ...auxDatasets, titleDataset]);
```

Comment:
`auxDatasets` should be in the first `usePrefetchValues` as they have the same dimension as `signal`.